cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -765,6 +769,25 @@ TYPED_TEST(MultiStepReductionTest, Mean)
             expected_value_nulls);
 }
 
+template <typename T>
+double calc_var(std::vector<T> const& v, int ddof, std::vector<bool> const& mask = {})
```

Comment:
```suggestion
double calc_var(cudf::host_span<T const> v, int ddof, cudf::host_span<bool const> mask = {})
```

Reply:
This bit broke template argument deduction, so will revert since it's just in a test.